exercises/practice/high-scores/.meta/lib/example.dart

```diff
@@ -0,0 +1,46 @@
+class HighScores {
+  List<int> scores = [];
```

Comment:
```suggestion
  List<int> scores = [];
```

I would have all references to scores or this.scores, changed to `_scores` or `this._scores`. An underscore makes the property `private` and we want that otherwise, the getter `getScores` does not make any sense.

Because the user/dev can pull the score values and redefine them at any time.